CoastalResilienceNetwork/GeositeFramework
# Attempt to support various underscore.js imports

## Overview
Adjust the use config to support all of the various ways that underscore
is imported within plugins.

The previous fix implemented in #1044 caused problems with plugins that
bring in their own version of underscore in via a CDN. The adjustments
here should make that possible while still allowing plugins to import
underscore via Use.js.

Also, rename the framework version of underscore to reduce confusion.

Connects to #669, #1043, #1046

### Notes

I'm not totally sure why this is working. It was arrived at through various tweaks of the Use config until various plugins that use underscore were all working.

## Testing Instructions

- Bring in regional-planning, flood-slr, and living-shorelines-explorer, and any other plugins that you know of that use underscore.
- Launch the framework, and ensure all of the plugins are working and that there are no errors related to underscore in the console (either errors finding underscore or undefined underscore methods.).
- Ensure regional-planning works before and after https://github.com/CoastalResilienceNetwork/regional-planning/pull/95. This will exercise two different approaches to using underscore.
